breaking: delegate dev static asset CORS handling to Vite (#16357)
sveltejs/kit: .changeset/ripe-trains-dance.md

---
'@sveltejs/kit': major
---

> breaking: delegate CORS handling to Vite for requests against the development server

Author: ```suggestion
breaking: delegate CORS handling to Vite for static directory requests during development
```
